dds_web/api/user.py

```diff
@@ -193,6 +198,17 @@ def invite_user(email, new_user_role, project=None):
 
         # Append invite to unit if applicable
         if new_invite.role in ["Unit Admin", "Unit Personnel"]:
+            # TODO Change / move this later. This is just so that we can add an initial unit admin.
+            if auth.current_user().role == "Super Admin":
+                if new_invite.role == "Unit Admin" and unit:
```

Comment:
Is it on purpose that you allow for _Unit Admin_ and _Unit Personnel_ in line 200, but now restrict to _Unit Admin_?